packages/indexer-agent/src/agent.ts

```diff
@@ -1008,6 +1012,9 @@ export class Agent {
     // Ensure all subgraphs in offchain subgraphs list are _always_ indexed
     for (const offchainSubgraph of this.offchainSubgraphs) {
       if (!deploymentInList(targetDeployments, offchainSubgraph)) {
+        logger.trace('Ensuring offchain subgraph is indexed', {
```

Comment:
This could potentially emit A LOT of log messages.  I'd rather keep the logs somewhat parsable.  
What about a single log that includes an array of subgraph ids?

Reply:
I think we can remove this, since offchain deployments are already logged in the `target` field of following logs